### PR TITLE
improve helm command

### PR DESCRIPTION
**Pull request by xw-patrick**
- remove --name  - this is not supported anymore in newer versions of helm
- use helm upgrade --install 
- add namespace

### CODE REVIEW

**Comment by nflaig** [comment]
@kreuzwerker-patrick I would recommend not using a namespace without a specific reason, this is also recommended by the [k8s docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#when-to-use-multiple-namespaces).

File: kubernetes/docs/try-with-minikube.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ npm run docker:build
 
 ```sh
 helm dependency build kubernetes/shopping-app
-helm install --name shopping-app --debug kubernetes/shopping-app
+helm upgrade --install shopping-app --debug kubernetes/shopping-app -n shopping-app --create-namespace
```
Comment (nflaig) [inline kubernetes/docs/try-with-minikube.md:80]:
```suggestion
helm upgrade --install shopping-app --debug kubernetes/shopping-app
```

Comment (kreuzwerker-patrick) [reply kubernetes/docs/try-with-minikube.md:80]: agreed - thanks for the review.
but removing --name
and using upgrade --install
makes sense, right?

Comment (nflaig) [reply kubernetes/docs/try-with-minikube.md:80]: removing `--name` definitly makes sense as it does not work with helm v3, not sure about `upgrade --install`, I personally prefer using it but someone new to helm might not understand what it does without reading the docs/help.

**Comment by dhmlau** [comment]
@kreuzwerker-patrick, thanks for the PR. Could you please also sign the DCO as well? Thanks.

**Comment by kreuzwerker-patrick** [comment]
> @kreuzwerker-patrick, thanks for the PR. Could you please also sign the DCO as well? Thanks.
@dhmlau - I tried to do so, but always get 403 back. 
remote: Permission to loopbackio/loopback4-example-shopping.git denied to kreuzwerker-patrick.
fatal: unable to access 'https://github.com/loopbackio/loopback4-example-shopping.git/': The requested URL returned error: 403

**Comment by mrmodise** [comment]
Thanks for the PR. If possible can we please bundle the 2 other PRs into this one?

**Comment by dhmlau** [comment]
@kreuzwerker-patrick,
> @dhmlau - I tried to do so, but always get 403 back.

From the error, it looks like you're trying to change it loopbackio/loopback4-example-shopping instead of from your fork.  Could you please try to amend the commit from your branch?

**Comment by mrmodise** [comment]
Closing due to inactivity. Feel free to open when ready. Thanks

**Pull request closed without merging**